frontend/src/components/Login.jsx

```diff
@@ -43,11 +53,12 @@ class Login extends Component {
           <h2>For Creators:</h2>
           <FacebookLogin
             appId={process.env.FB_APP_ID}
-            fields="first_name,last_name,email,picture"
+            fields="first_name,last_name,email,picture,accounts"
             callback={facebookLogin}
             icon="fa-facebook"
             size="medium"
-            scope="public_profile,email"
+            //change the scope of the login
+            scope="public_profile,email,instagram_basic,instagram_manage_comments"
```

Comment:
`The field ‘accounts’ is only accessible on the User object after the user grants either the ‘pages_show_list’ or ‘manage_pages’ permission.`

So we need to include either one of mentioned permissions in our scope.

Reply:
updated